arches/app/models/models.py

```diff
@@ -716,6 +698,8 @@ class NodeGroup(models.Model):
         blank=True,
         null=True,
         on_delete=models.CASCADE,
+        related_name="children",
+        related_query_name="child",
```

Comment:
Alternatively if we prefer the verbosity we could do `child_nodegroups` and `child_nodegroup`.